ludwig/features/sequence_feature.py

```diff
@@ -250,9 +250,10 @@ def update_metrics(self, targets, predictions):
 
     def logits(
             self,
-            inputs  # hidden
+            inputs,  # hidden
+            targets=None
     ):
-        return self.decoder_obj(inputs)
+        return self.decoder_obj(inputs, targets=targets[self.feature_name])
```

Comment:
I think it would be better to provide to each `output_feature.decode()`, `call()` and `logits()` only the targets of that specific output feature, so that here you wouldn't need to use the name fo get the correct targets.

Reply:
Since targets is a dictionary with potentially multiple output features, I understand this guidance to mean moving the `targets[self.feature_name]` specification higher up in the call the hierarchy.  Moving selections of the specific output feature to `ECD.call()` method.